src/commands/status/services.js

```diff
@@ -39,13 +39,14 @@ class ServicesStatusCommand extends BaseCommand {
     }
 
     const tableRows = [
-      ['Service', 'Container ID', 'Version', 'Status'],
+      ['Service', 'Container ID', 'Image', 'Version', 'Status'],
```

Comment:
It's not actually version, it's a tag. It could be anything. I would just keep them together in the image column. 

P.S. Probably we should combine them in configs as well.

Reply:
OK, done. Let's change the configs in a separate PR together with the networking fixes.

Reply:
I would separate them into two to make them more atomic